packages/dashmate/src/commands/group/reset.js

```diff
@@ -69,7 +71,14 @@ class GroupResetCommand extends GroupBaseCommand {
                 config.set('core.masternode.operator.privateKey', baseConfig.core.masternode.operator.privateKey);
               }
 
-              return resetNodeTask(config);
+              return new Listr([
+                {
+                  task: () => resetNodeTask(config),
+                },
+                {
+                  task: () => generateSystemDataContractKeysTask(config, config.get('network')),
```

Comment:
Did you test it? I think it won't work because all configs will have different keys now so init chain won't be deterministic. If you see how it worked before. We generated it once on setup and set to all configs. You should do the same